golden-tests/dhall-to-cabal/map-source-repo.cabal

```diff
@@ -0,0 +1,17 @@
+name: repo
```

Comment:
I think what we actually really want is a test that `mapSourceRepo` produces some normalized Dhall, but it might be a little out of scope for this PR. If you want to add a new section to the golden tests (that basically golden test one Dhall file normalizes to another), I'd be on board with that. If not, it can be done later.

Reply:
That section would be usefult to test other prelude functions in isolation (among other things), right?
I agree it would be usefult but if you dont mind i would prefer to add it in another pr.

Reply:
Another PR is good with me!